include/nbl/builtin/hlsl/spirv_intrinsics/glsl.std.450.hlsl

```diff
@@ -0,0 +1,23 @@
+#ifndef _NBL_BUILTIN_HLSL_SPIRV_INTRINSICS_GLSL_STD_450_INCLUDED_
+#define _NBL_BUILTIN_HLSL_SPIRV_INTRINSICS_GLSL_STD_450_INCLUDED_
+
+namespace nbl 
+{
+namespace hlsl
+{
+namespace spirv
+{
+
+//[[vk::ext_instruction(73 /*FindILsb*/, "GLSL.std.450")]]
+//int FindILsb(int32_t val);
+
+[[vk::ext_instruction(74 /*FindUMsb*/, "GLSL.std.450")]]
+int FindSMsb(int32_t val);
+
+[[vk::ext_instruction(75 /*FindUMsb*/, "GLSL.std.450")]]
```

Comment:
see what the other `spirv_intrinsics` header does, include the `unified1.hpp` please and use the enums instead of integer literals.

Also use `int32_t` instead of `int` for clarity as the return types